docs/eddie.md

```diff
@@ -0,0 +1,101 @@
+# nf-core/configs: Eddie Configuration
+
+nf-core pipelines sarek, rnaseq, and atacseq have all been tested on the University of Edinburgh Eddie HPC.
+
+## Getting help
+
+There is a Slack channel dedicated to eddie users on the MRC IGMM Slack: [https://igmm.slack.com/channels/eddie3](https://igmm.slack.com/channels/eddie3)
+
+## Using the Eddie config profile
+
+To use, run the pipeline with `-profile eddie` (one hyphen).
+This will download and launch the [`eddie.config`](../conf/eddie.config) which has been pre-configured with a setup suitable for the [University of Edinburgh Eddie HPC](https://www.ed.ac.uk/information-services/research-support/research-computing/ecdf/high-performance-computing).
+
+The configuration file supports running nf-core pipelines with Docker containers running under Singularity by default. Conda is not currently supported.
+
+```bash
+nextflow run nf-core/PIPELINE -profile eddie  # ...rest of pipeline flags
+```
+
+Before running the pipeline you will need to install Nextflow or load it from the module system. Generally the most recent version will be the one you want. 
+If you want to run a Nextflow pipeline that is based on [DSL2](https://www.nextflow.io/docs/latest/dsl2.html), you will need a version that ends with '-edge'.
+
+To list versions:
+
+```bash
+module avail igmm/apps/nextflow
+```
+
+To load the most recent version:
+```bash
+module load igmm/apps/nextflow
+```
+
+This config enables Nextflow to manage the pipeline jobs via the SGE job scheduler and using Singularity for software management.
+
+## Singularity set-up
+
+Load Singularity from the module system and, if you have access to `/exports/igmm/eddie/NextGenResources`, set the Singularity cache directory to the NextGenResources path for the pipeline and version you want to run. If this does not exist, please contact the [IGMM Data Manager](data.manager@igmm.ed.ac.uk) to have it added. You can add these lines to the file `$HOME/.bashrc`, or you can run these commands before you run an nf-core pipeline.
+
+If you do not have access to `/exports/igmm/eddie/NextGenResources`, set the Singularity cache directory to somewhere sensible that is not in your `$HOME` area (which has limited space). It will take time to download all the Singularity containers, but you can use this again.
+
+```bash
+module load singularity
+export NXF_SINGULARITY_CACHEDIR="/exports/igmm/eddie/NextGenResources/nextflow/singularity/nf-core-rnaseq_v3.0"
```

Comment:
FYI, the latest release(s) of `nf-core tools` now has nice integration with this which may be useful to you: https://nf-co.re/tools#singularity-cache-directory

Reply:
A shared cache is what you want then - same situation here, so advanced users ( = core unit team) can run a pipeline once for testing purposes and the containers will then be available to everyone so should run smoothly. Bonus: if something goes wrong, these more experienced users can take care of the issue immediately which is better than other users who might have more issues debugging :-)

Reply:
Yup, agree - shared should be best. Especially with DSL2 where there will be lots of containers and hopefully a lot will be shared between pipelines. The filenames are based on the exact version used so should be no issue with reproducibility.